packages/fast-usdc/src/types.ts

```diff
@@ -58,11 +58,14 @@ export interface PoolMetrics extends PoolStats {
 }
 
 export interface ChainPolicy {
-  nobleContractAddress: EvmHash;
+  /** `msg.sender` of DepositAndBurn to TokenMessenger must be an attenuated wrapper contract that does not contain `replaceDepositForBurn` */
```

Comment:
👌

Reply:
I'd like to see a few words from this JSDoc lifed to the PR description and/or title.

not critical.